scripts/patrol.py

```diff
@@ -638,6 +639,12 @@ def filter_patrols(self, possible_patrols, biome, patrol_size, current_season, p
             if patrol.season not in [current_season, "Any"]:
                 continue
 
+            # filtering for dates
+            if 'date_exclusive' in patrol.tags:
```

Comment:
Would it be possible to remove the "date_exclusive" tag, and only check for the special date tags? The "date_exclusive" tags appears redundant.